dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -810,7 +811,7 @@ public void handleInfo(String entryPath) {
                     description = "";
                 }
 
-                String author = container.getAuthor();
```

Comment:
Based on the discussion, weren't you going to change this and call `getAuthor()` after all?

Reply:
No, I thought the idea was that the 1.15 webservice would have both. 

We would make the 1.15 cli use 'authors' instead of author, but both the 1.14 and 1.15 cli would thus work with the 1.15 webservice. Then in 1.16 when we remove `author`, we're hoping that users would be using the 1.15 cli and thus not notice anything.